cmd/kubectl-datadog/find/find.go

```diff
@@ -0,0 +1,115 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package find
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/DataDog/datadog-operator/pkg/plugin/common"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var (
+	findExample = `
+  # find the datadog agent pod monitoring a pod named foo
+  %[1]s find foo
+`
+)
+
+// options provides information required by Datadog find command
+type options struct {
+	genericclioptions.IOStreams
+	common.Options
+	args    []string
+	podName string
+}
+
+// newOptions provides an instance of getOptions with default values
+func newOptions(streams genericclioptions.IOStreams) *options {
+	o := &options{
+		IOStreams: streams,
+	}
+	o.SetConfigFlags()
+	return o
+}
+
+// New provides a cobra command wrapping options for "find" sub command
+func New(streams genericclioptions.IOStreams) *cobra.Command {
+	o := newOptions(streams)
+	cmd := &cobra.Command{
+		Use:          "find [pod name] [flags]",
```

Comment:
What do you think if we move this command as a `agent` subcommand ?
example: `kubectl datadog agent find <myapp-pod>`